hw/top_earlgrey/formal/top_earlgrey_fpv_ip_cfgs.hjson

```diff
@@ -53,6 +53,7 @@
                import_cfgs: ["{proj_root}/hw/formal/tools/dvsim/common_fpv_cfg.hjson"]
                rel_path: "hw/top_earlgrey/ip_autogen/rv_plic/{sub_flow}/{tool}"
                cov: true
+               exp_fail_hjson: "{proj_root}/hw/top_earlgrey/ip_autogen/fpv/rv_plic_expected_failure.hjson"
```

Comment:
This is just an idea (I let @sriyerg decide which approach would be better):

How about we just define an `exp_sva_fail` dict here in the `*_cfgs.hjson`?

E.g.:
```
exp_sva_fail: {
  unreachable:
  [
    rv_plic_tb.dut.FpvSecCmRegWeOnehotCheck_A:precondition1
    rv_plic_tb.dut.u_reg.u_prim_reg_we_check.u_prim_onehot_check.Onehot0Check_A:precondition1
    rv_plic_tb.dut.u_reg.u_prim_reg_we_check.u_prim_onehot_check.gen_enable_check.gen_not_strict.EnableCheck_A:precondition1
  ]
}
```

We could then rely on the `import_cfgs` feature if we wanted to split this out into a separate hjson file, instead of having to add another file parser in dvsim.

Just a thought, we can also leave it as is.

Reply:
Thanks @msfschaffner! That is a great idea.

Here is my small concern:
FPV uses the batch script, so all the IPs are listed together. If the list of failures are very long, it might affect the readability for the `*_sim_cfgs.hjson` file.

Reply:
Ah yes but in that case I imagined we could just move the definition to a `cfgs.hjson` that can be imported with `import_cfgs`. I.e., the definition does not necessarily have to be in the batch script. 
Note that it is possible to list imported configs in the batch script as well here:
https://github.com/lowRISC/opentitan/blob/063cd66859ad4f31db0511729c71b5c19d68779c/hw/top_earlgrey/formal/top_earlgrey_fpv_sec_cm_cfgs.hjson#L28

Reply:
I see thanks Michael.
I have a followup question - 
The current parse is mainly done in the `parse-formal-report.py`. Parsing the results and summary are not really done in dvsim.
If I understand your suggestion correctly, we will need to feed this `dict` in `parse-formal-report.py` right?

Reply:
Ah I see what you mean.
Yeah in that case it may indeed be simpler to just pass in a file path that can be forwarded to the parser script.
Nevermind then.